src/settings/mod.rs

```diff
@@ -4,25 +4,71 @@
 
 //! Application settings.
 
-use std::env;
+use std::{
+    env,
+    fmt::{self, Display},
+};
 
 use config::{Config, ConfigError, Environment, File};
+use serde::de::{Deserialize, Deserializer, Error, Unexpected};
 
-use deserialize;
+use duration::Duration;
 use logging::MozlogLogger;
+use send::EmailAddress;
```

Comment:
It's fine for now, no need to change it, but fwiw at some point in the future I might pull `EmailAddress` out to its own module. Something about `settings` depending on `send` feels the wrong way round to me, even though it doesn't actually matter really.